day01/Husky/src/components/KeyButton.tsx

```diff
@@ -0,0 +1,18 @@
+import Key from "../models/Key";
+
+export default function KeyButton(props: Key) {
+  return (
+    <div
+      className={
+        "border-4 border-solid border-black rounded-lg m-4 text-2xl py-4 px-2 w-40 " +
```

Comment:
why not using `clsx`, or write a utility for compose class name

Reply:
fixed